pkg/kubelet/kubelet_pods.go

```diff
@@ -1087,7 +1086,6 @@ func (kl *Kubelet) HandlePodCleanups() error {
 		desiredPods[pod.UID] = sets.Empty{}
 	}
 	// Stop the workers for no-longer existing pods.
-	// TODO: is here the best place to forget pod workers?
```

Comment:
this is where we centralize cleanup for all non-desired pods, so there isnt really a better place to do this now.